xml/runtime_config.xml

```diff
@@ -10,60 +10,259 @@
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
    <dm:translation>yes</dm:translation>
-   <dm:release>SES 6</dm:release>
+   <dm:release>SES 7</dm:release>
   </dm:docmanager>
  </info>
  <para>
-  This chapter provides a list of important &ceph; cluster settings and their
-  description. The settings are sorted by topic.
+  This chapter describes how to configure the &ceph; cluster by means of
+  configuration options.
  </para>
- <sect1 xml:id="ceph-config-runtime">
-  <title>Runtime Configuration</title>
+ <sect1 xml:id="cha-ceph-configuration-ceph-conf">
+  <title>The <filename>ceph.conf</filename> File</title>
 
   <para>
-   The actual cluster behavior is determined not by the current state of the
-   <filename>ceph.conf</filename> file but by the configuration of the running
-   &ceph; daemons, which is stored in memory.
+   &cephadm; uses a minimized <filename>ceph.conf</filename> file that only
```

Comment:
I think "minimized" is not a good word here. Perhaps just "very simple" or "very small".